app-android/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("DSL_SCOPE_VIOLATION")
```

Comment:
Why do we need this?

Reply:
I had an warning from AS. https://github.com/gradle/gradle/issues/18107
Is there another way without suppressing?

Reply:
I'm usually just ignoring it, but let's keep it.